data/transactions/logic/eval_test.go

```diff
@@ -1255,6 +1259,7 @@ func TestGlobal(t *testing.T) {
 		8:  {CallerApplicationAddress, globalV8TestProgram},
 		9:  {CallerApplicationAddress, globalV9TestProgram},
 		10: {AssetOptInMinBalance, globalV10TestProgram},
+		11: {AssetOptInMinBalance, globalV11TestProgram},
```

Comment:
I feel like I'm seeing a bug that's my fault. We did add another global, `GenesisHash` so I think that should be here.  I'll look more closely in master.

Reply:
I made a PR to fix this in master. It's just a lack of a test, so there's not reason to make any changes here.